twitter/twitter.php

```diff
@@ -75,7 +75,6 @@
 use Friendica\Model\Group;
 use Friendica\Model\Item;
 use Friendica\Model\ItemContent;
-use Friendica\Model\Photo;
```

Comment:
Photo is not needed anymore? Just asking as it has nothing to do with fixing that PHP warning at the first glance for me.

Reply:
A search for it hasn't found anything.

Reply:
My IDE checks for `use` usage and warns me if a statement isn't used anymore in the file. It just is a basic cleanup task.